locales/en.yml

```diff
@@ -70,3 +70,22 @@ en:
       description: Discover the story of our Logo
       page_title: Logo
     title: Logo of Ruby Nord
+  talks:
+    description: |
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```

Comment:
OK make sense, but I have no idea how to translate that in french.

Moreover, what's the difference between `all` and `beginner`? I think I don't understand something.

For example:  Why `Présentation de la gem administrate` is `all` whereas `Craft everything with Middleman` is `beginner`?